src/i18n/locales/en/translation.json

```diff
@@ -58,41 +61,68 @@
         }
       }
     },
-    "choose_connector": {
-      "title": {
-        "welcome": "Do you have a lightning wallet?",
-        "options": "Add a new lightning account"
-      },
-      "description": "You need to first connect to a lightning wallet so that you can interact with your favorite websites that accept bitcoin lightning payments!",
+    "choose_path": {
+      "title": "Create or Connect Wallet",
+      "description": "Create a new Alby Wallet, log in to an existing one or connect to an external wallet or node",
       "alby": {
         "title": "Alby Wallet",
-        "description": "Create or log in to your Alby account",
-        "pre_connect": {
-          "title": "Your Alby Lightning Wallet",
-          "login_account": "Create or log in to your Alby account.",
-          "host_wallet": "We host a Lightning wallet for you!",
-          "email": {
-            "label": "Email Address"
-          },
-          "optional_lightning_note": {
-            "part1": "Your Alby account also comes with an optional",
-            "part2": "Lightning Address",
-            "part3": ". This is a simple way for anyone to send you Bitcoin on the Lightning Network.",
-            "part4": "learn more"
+        "description": "Create or log in to your Alby Wallet",
+        "create_new": "Create new"
+      },
+      "other": {
+        "title": "Other Wallets",
+        "description": "Connect to your external lightning wallet or node",
+        "and_more": "and more...",
+        "connect": "Connect to Lightning Wallet"
+      }
+    },
+    "alby": {
+      "pre_connect": {
+        "set_password": {
+          "choose_password": {
+            "label": "Password"
           },
-          "optional_lightning_address": {
-            "label": "Choose your Lightning Address (optional)",
-            "suffix": "@getalby.com",
-            "title": "numbers and letters, at least 3 characters"
+          "confirm_password": {
+            "label": "Confirm Password"
           },
           "errors": {
-            "create_wallet_error": "Failed to log in or create a new account. If you need help, please contact support@getalby.com"
+            "enter_password": "Please enter a password.",
+            "confirm_password": "Please confirm your password.",
+            "mismatched_password": "Passwords don't match."
           }
+        },
+        "title": "Your Alby Wallet",
+        "create_account": "Create your Lightning Wallet hosted by Alby.\nYou can always regain access to Alby Wallet using these credentials.",
```

Comment:
FYI: we recently changed all copy from "Lightning" / "Bitcoin" to "lighting" / "bitcoin" on Alby content pages. You will still find them capitalized in some headlines but those should be the only exceptions. We haven't yet discussed how to handle "the Alby Wallet" / "the Alby Lightning Wallet", which are both new to me :)

Reply:
@jankoegel thanks for letting me know. I have updated the ones I think should be lowercase.